packages/py-moose-lib/moose_lib/dmv2/registry.py

```
@@ -137,6 +138,16 @@ def get_view(name: str) -> Optional["View"]:
     return _views.get(name)
 
 
+def get_olap_dictionaries() -> dict:
+    """Get all registered OLAP dictionaries."""
+    return _olap_dictionaries
+
+
+def get_olap_dictionary(name: str) -> Optional[Any]:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `get_olap_dictionary`

(ANN401)